Documentation/tutorials/data-prep-tutorial.md

```diff
@@ -0,0 +1,211 @@
+# Map track event data with Data Prep for Data Collection <!-- omit in toc -->
+
+## Table of Contents <!-- omit in toc -->
+- [Overview](#overview)
+  - [Environment](#environment)
+  - [Prerequisites](#prerequisites)
+- [Data Prep for Data Collection mapping](#data-prep-for-data-collection-mapping)
+  - [Mapping custom `contextData` keys](#mapping-custom-contextdata-keys)
+  - [Mapping calculated fields using Analytics functions](#mapping-calculated-fields-using-analytics-functions)
+- [Validating Data Prep mapping using Assurance](#validating-data-prep-mapping-using-assurance)
+
+## Overview
+This tutorial covers how to use [Data Prep for Data Collection](https://experienceleague.adobe.com/en/docs/experience-platform/datastreams/data-prep) to enable using Edge Bridge event data in other Experience Platform applications, by mapping event data to the [Experience Data Model (XDM)](https://experienceleague.adobe.com/docs/experience-platform/xdm/home.html) specification.
+
+```mermaid
+graph TD;
+    step1(1<br/>Existing app using Adobe Analytics) -->
+    step2(2<br/>Adobe Experience Platform<br/>Update server-side configuration) --> 
+    step3(3<br/>Edge Bridge<br/>Update app to send event data via Edge Network to Analytics) --> 
+    step4(4<br/>Assurance<br/>Verify implementation) --> 
+    step5("5<br/>Data mapper<br/>Map event data to Experience Data Model (XDM) format") -->
+    step6(6<br/>Assurance<br/>Verify event data to XDM conversion);
+
+style step5 fill:#3273de,stroke:#333,stroke-width:2px,color:#fff
+style step6 fill:#3273de,stroke:#333,stroke-width:2px,color:#fff
+```
+
+### Environment
+- Android Studio version which supports Gradle plugin 8.2.
+- Android emulator or device.
```

Comment:
is this the only change compared to iOS or anything else I should be paying more attention while reviewing?